### PR TITLE
Add MSA logging to test app

### PR DESCRIPTION
**Pull request by MatkovIvan**

### CODE REVIEW

File: apps/sasquatch/src/projectDependency/java/com/microsoft/appcenter/sasquatch/activities/MicrosoftAuthenticationActivity.java [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+                        authenticationResult.getUser().getUserIdentifier(),
+                        authenticationResult.getAccessToken());
+            }
+
+            @Override
+            public void onError(MsalException exception) {
+                Log.e(LOG_TAG, exception.getMessage());
+                if (exception instanceof MsalUiRequiredException) {
+
+                    // This explicitly indicates that developer needs to prompt the user, it could be refresh token is expired, revoked
```
Comment (guperrot) [inline apps/sasquatch/src/projectDependency/java/com/microsoft/appcenter/sasquatch/activities/MicrosoftAuthenticationActivity.java]: Please use `/*` comment.

File: apps/sasquatch/src/projectDependency/java/com/microsoft/appcenter/sasquatch/activities/MicrosoftAuthenticationActivity.java [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+import com.microsoft.appcenter.sasquatch.R;
+import com.microsoft.identity.client.AuthenticationCallback;
+import com.microsoft.identity.client.AuthenticationResult;
+import com.microsoft.identity.client.MsalException;
+import com.microsoft.identity.client.MsalUiRequiredException;
+import com.microsoft.identity.client.PublicClientApplication;
+
+public class MicrosoftAuthenticationActivity extends AppCompatActivity {
+
+    private static final String LOG_TAG = "MicrosoftAuthentication";
```
Comment (guperrot) [inline apps/sasquatch/src/projectDependency/java/com/microsoft/appcenter/sasquatch/activities/MicrosoftAuthenticationActivity.java]: Please add blank lines between fields.

File: apps/sasquatch/src/projectDependency/java/com/microsoft/appcenter/sasquatch/activities/MicrosoftAuthenticationActivity.java [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+    private static final String[] SCOPES = new String[]{"User.Read"};
+
+    private PublicClientApplication mApplication;
+    private AuthenticationResult mAuthentication;
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_msa_auth);
+
```
Comment (guperrot) [inline apps/sasquatch/src/projectDependency/java/com/microsoft/appcenter/sasquatch/activities/MicrosoftAuthenticationActivity.java]: Please remove either add a comment or remove blank line.

File: apps/sasquatch/src/jcenterDependency/java/com/microsoft/appcenter/sasquatch/activities/MicrosoftAuthenticationActivity.java [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+        setContentView(R.layout.activity_msa_auth);
+    }
+
+    public void onLoginClick(View view) {
+    }
+
+    public void onRefreshClick(View view) {
+    }
+}
+
```
Comment (dhei) [inline apps/sasquatch/src/jcenterDependency/java/com/microsoft/appcenter/sasquatch/activities/MicrosoftAuthenticationActivity.java]: nit - remove extra new line

**Comment by guperrot** [comment]
Keeping this PR for later use when we will do AAD.

**Pull request closed without merging**